### ISSUE TITLE
Help on Installation steps 

### ISSUE THREAD

**Issue by baalasangar** [ngageoint/gamification-server#12]
Installation steps are at very high level. Is there any detailed installation steps and usage  guide like how to configure add a new project . We do not find the hardware recommendation or OS recommendations Can you please share as the same ?

**Comment by Edward-pantoja** [comment]
I second this. I find the installation instructions hard to follow since I am such a newbie. A more in depth instruction would be so helpful. Thank you for all that you are doing. Eddie
